docs/Software Architecture Document/Component Interaction View/assets/plugin_manager_interaction_view.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+
+participant "Protocol handler" as PH
+participant "Application Manager" as AM
+participant "Plugin Manager" as PM
+participant "Plugin 1" as P1
+participant "Plugin 2" as P2
+
+PH -> AM : Send data
+AM -> AM : Create message from data
```

Comment:
I would suggest to extent RPC diagram.
Because at this moment Plugin Manger is a Command Plugin only.